src/kubectl.ts

```diff
@@ -147,16 +153,35 @@ async function kubectllInternalArgs(args: string[], namespace: string | undefine
 }
 
 export async function baseKubectlPath(): Promise<string> {
-	let result : shell.FindBinaryResult = await shell.findBinary('kubectl');
+	const result : FindBinaryResult = await findBinary('kubectl');
```

Comment:
I think you had an issue with conflict resolution during rebase. This part is available in the shell class